clients/client-java/src/main/java/org/apache/gravitino/client/GravitinoClient.java

```diff
@@ -84,6 +84,24 @@ private GravitinoClient(
     this.metalake = loadMetalake(metalakeName);
   }
 
+  /**
+   * Constructs a new GravitinoClient with the given URI, authenticator and AuthDataProvider.
+   *
+   * @param uri The base URI for the Gravitino API.
+   * @param authDataProvider The provider of the data which is used for authentication.
+   * @param checkVersion Whether to check the version of the Gravitino server. Gravitino does not
+   *     support the case that the client-side version is higher than the server-side version.
+   * @param headers The base header for Gravitino API.
+   * @throws NoSuchMetalakeException if the metalake with specified name does not exist.
+   */
+  private GravitinoClient(
```

Comment:
Are these expected changes in this PR?

Reply:
No, anything other than the document can be ignored

Reply:
Do I need to close PR, fork the repository again, make modifications based on this repository, and submit PR again

Reply:
@lsyulong 
You can remove this unrelated code directly and do not need to close this PR.